sql/mysqld.cc

```diff
@@ -1966,6 +1965,7 @@ static void clean_up(bool print_message)
   logger.cleanup_base();
 
   injector::free_instance();
+  mysql_bin_log.stop_background_thread();
```

Comment:
Is this needed? Because `mysql_bin_log.cleanup()` calls `stop_background_thread()` towards its start.
```
  ...
  if (inited)
  {
    xid_count_per_binlog *b;

    /* Wait for the binlog background thread to stop. */
    if (!is_relay_log)
      stop_background_thread();
   ...
```